### ISSUE TITLE
Keyboard shortcuts 🩳

### ISSUE THREAD

**Issue by MusicFreak456** [WizzWriters/spelldraw-app#141]
Please add ideas for the most useful shortcuts to implement.

**Comment by ventus550** [comment]
1. Switching between tools
- move (M or wheel press and hold, the last one might be tricky)
- eraser (E)
- pencil (P)
2. Presets (some random examples)
- some basic colors (ctrl + <number key>)
- stroke size (not sure what to bind it to though)
3. Future work?
- jump to someone's mouse pointer location (would make it easier to find people)
- instant magic / no correction wait time (alt hold?)

**Comment by ventus550** [comment]
Also might be nice to add the shortcuts to the hover tooltips. This way people can easily figure out the basic keybinding.

**Comment by MusicFreak456** [comment]
> Also, might be nice to add the shortcuts to the hover tooltips. This way people can easily figure out the basic keybinding.

Yes, beside adding the tooltips, I was thinking we can add some cheet sheet to display (for example under "help" in navbar) where we can list most important shortcuts and also showcase the correction functionalities (with GIFs or something)

**Comment by ventus550** [comment]
> > Also, might be nice to add the shortcuts to the hover tooltips. This way people can easily figure out the basic keybinding.
> 
> Yes, beside adding the tooltips, I was thinking we can add some cheet sheet to display (for example under "help" in navbar) where we can list most important shortcuts and also showcase the correction functionalities (with GIFs or something)

I would leave that for a settings page where one could also change the keybindings.

**Comment by MusicFreak456** [comment]
Nah, if we bury it too deep, nobody will find it. I think having like '?' symbol somewhere in the interface that will display some concise info about features of the board will be nice.
